__tests__/override-angular-versions.test.ts

```diff
@@ -19,7 +19,7 @@ describe(overrideAngularVersions.name, () => {
       (v11_1.devDependencies as any) = {};
 
       const actualPackageJson = overrideAngularVersions({
-        projectVersions: fullPackageJson,
+        projectVersions: fullPackageJson as any,
```

Comment:
Because our `PackageJson` types only consider Angular-related dependencies. The `fullPackageJson` is a superset type, not a subset, so `Partial` won't help.

Reply:
What about something like `type PackageJsonExtended = PackageJson & {[key: string] => unknown}`